rules/falco_rules.yaml

```diff
@@ -3398,6 +3398,29 @@
   priority: WARNING
   tags: [host, container, mitre_execution, mitre_defense_evasion, T1036.005, T1059.004]
 
+#The given rule is designed to detect suspicious activities related to disabling system recovery features. 
+#The rule looks for specific commands being executed such as rm -rf, chattr +i, mkfs.ext4 -E nodiscard, mount remount,ro, 
+#and disabling certain services such as systemd-backlight@.service, apport.service, rescue.service, emergency.service, and recovery-mode.service. 
+#https://attack.mitre.org/techniques/T1490/
+#Mitre Impact: Inhibit system recovery
+
+- rule: Disable Recovery Features
+  desc: Detects disabling system recovery features by deleting or disabling services and commands.
+  condition: >
+      (spawned_process and proc.name = "rm" and proc.args contains "-rf") or
```

Comment:
Some initial feedback: 

- We can use `spawned_process` first and then do the conditioning afterwards in a `spawned_process and ( () or () or () )`. This will reduce the AST expression and since the rules push down is by event type the overall rules evaluation will become more clear.
- Is this detections intended to run on the host only?
- Could some conditions around the file system be based on respective syscalls for increased robustness vs spawned process and command args?
- In the `output` we need the output fields, see existing rules for guidance, we can discuss which fields are most relevant.

Reply:
@incertum  Sure.